source/stylesheets/base/_variables.scss

```diff
@@ -1,18 +1,29 @@
 // Breakpoints
-$breakpoint-medium: 480px;
-$breakpoint-large: 890px;
+$breakpoint-medium: 40em;
+$breakpoint-large: 55em;
 
 // Colors
-$teal: #12939A;
+$teal: #118288;
 $teal-dark: mix($teal, black, 80%);
```

Comment:
love it!

Reply:
Thanks - now passes WCAG AA (not AAA though)